SimpleWAWS/Code/BackgroundQueueManager.cs

```diff
@@ -172,6 +172,11 @@ private void OnTimerElapsed(object state)
             }
         }
 
+        private List<ResourceGroup> GetResourceGroupsWithHighStorageUsage()
+        {
+            return null;
```

Comment:
Not needed anymore since we wont monitor storage usage. Only a 1 hour window will be available without possibility to extend. Will remove it